src/test/java/com/wikia/webdriver/testcases/mercurytests/SearchTests.java

```diff
@@ -106,6 +148,7 @@ public void mercury_search_searchInputDoesNotCoverNavigation() {
     Assertion.assertFalse(resultsPage.getSearch().isSearchInputFieldEditable());
   }
 
+  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
   @Test(groups = "mercury_search_searchNoResultsPageDisplayed")
   public void mercury_search_searchNoResultsPageDisplayed() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)